File: opcua-webservice-spring/core/src/main/java/org/opcfoundation/webserver/service/message/reactiveobject/ReadObjectAttributeResponse.java
package org.opcfoundation.webserver.service.message.reactiveobject;
import org.eclipse.milo.opcua.stack.core.types.builtin.LocalizedText;
import org.eclipse.milo.opcua.stack.core.types.builtin.unsigned.UByte;
import org.opcfoundation.webserver.types.common.UaObjectId;

public class ReadObjectAttributeResponse {
    private final String browseName;
    private final LocalizedText displayName;
    private final LocalizedText description;
    private final UByte eventNotifier;
    // To add writeMask

    public ReadObjectAttributeResponse(
            UaObjectId objectId,
            LocalizedText displayName,
            LocalizedText description)
    {
        this(objectId,displayName,description, UByte.valueOf(0));
    }

    public ReadObjectAttributeResponse(
            UaObjectId objectId,
            LocalizedText displayName,
            LocalizedText description,
            UByte eventNotifier)
    {
        this.browseName = (null == objectId.getInstance()) ? objectId.getId() : objectId.getInstance().browseName();
        this.displayName = displayName;
        this.description = description;
        this.eventNotifier = eventNotifier;
    }

    public String getBrowseName() {
        return browseName;
    }

    public LocalizedText getDisplayName() {
        return displayName;
    }

    public LocalizedText getDescription()
    {
        return description;
    }

    public UByte getEventNotifier() {
        return eventNotifier;
    }
}

File: opcua-webservice-node/source/src/server/service/message/reactiveobject/ReadObjectAttributeResponse.ts
import { UaLocalizedText } from "opcua-webapi-ts";

export class ReadObjectAttributeResponse {
    private readonly _browseName: string;
    private readonly _displayName: UaLocalizedText;
    private readonly _description: UaLocalizedText;
    private readonly _eventNotifier: number;
    private readonly _writeMask: number;

    constructor(
        browseName: string,
        displayName: UaLocalizedText,
        description: UaLocalizedText,
        eventNotifier: number = 0,
        writeMask: number = 0) {
        this._browseName = browseName;
        this._displayName = displayName;
        this._description = description;
        this._eventNotifier = eventNotifier;
        this._writeMask = writeMask;
    }

    get browseName(): string {
        return this._browseName;
    }

    get displayName(): UaLocalizedText {
        return this._displayName;
    }

    get writeMask(): number {
        return this._writeMask;
    }

    get description(): UaLocalizedText {
        return this._description;
    }

    get eventNotifier(): number {
        return this._eventNotifier;
    }
}